public/translations/locale-en.json

```diff
@@ -91,7 +91,8 @@
         "HOW_MUCH_DO_MEMBERS_PAY": "How much do {{members}} pay at {{address}}?",
         "HOW_MUCH_DO_YOU_AND_MEMBERS_PAY": "How much do {{members}} pay at {{address}}?",
         "HOW_MUCH_DO_YOU_PAY": "How much do You pay at {{address}}?",
-        "TITLE": "How much {{members}} pay in monthly rent?"
+        "TITLE_HOUSEHOLD": "How much does {{members}} pay in monthly rent?",
+        "TITLE_YOU": "How much do {{members}} pay in monthly rent?"
```

Comment:
@chrisdolendo I guess I'm not seeing why `{{members}}` really needs to be a variable for these ones. Isn't `TITLE_YOU` _always_ `"How much do you pay in monthly rent?"` ? And I think that was my point on the original story, it will always be easier for a translator to translate: `How much do you pay in monthly rent?` rather than translate `How much do {{members}} pay in monthly rent?` and then separately translate the word `You`, but knowing that it's the version of `You` that grammatically fits into that first sentence as `{{members}}`.